HISTORY.rst

```diff
@@ -8,6 +8,7 @@ History
 
 * ``setuptools`` was incorrectly listed as a runtime dependency. This has
   been removed.
+* Added support for the new risk reasons outputs in minFraud Factors.
```

Comment:
The changelog entry for the various libraries should be mostly similar. You can see what I used [in the .NET library PR](https://github.com/maxmind/minfraud-api-node/pull/1415) (although I will probably update it a bit as mentioned in the comment).

> Note that this is a beta release and the change log file should reflect that in both the version it lists as well as the change description itself.

We'll want the version to reflect that this is a beta release. Further, I think we'd want to bump the minor version.

Reply:
> bump the minor version

The latest version is [v2.11.0](https://github.com/maxmind/minfraud-api-python/releases/tag/v2.11.0), so the changelog already has a version bump.

Reply:
It currently has a patch version bump, not a minor version bump

Reply:
You're right, updated 👍